README.md

```diff
@@ -21,8 +21,12 @@ Our script does the following:
 - Copies used images to an output folder, to allow them to be moved to a new home.
 - Afterwards, it asks if you want to try downloading the original size images using [download_better_images.py](https://raw.githubusercontent.com/timhutton/twitter-archive-parser/main/download_better_images.py).
```

Comment:
Please insert a comment to say that it asks if you additionally want to output html.